src/hyperion/experiment_plans/flyscan_xray_centre_plan.py

```diff
@@ -177,13 +179,15 @@ def kickoff_and_complete_gridscan(
     eiger: EigerDetector,
     synchrotron: Synchrotron,
     zocalo_environment: str,
+    scan_points: List[AxesPoints[Axis]],
 ):
     @TRACER.start_as_current_span(DO_FGS)
     @bpp.set_run_key_decorator(DO_FGS)
     @bpp.run_decorator(
         md={
             "subplan_name": DO_FGS,
             "zocalo_environment": zocalo_environment,
+            "scan_points": scan_points,
```

Comment:
On second thought I actually don't like this in the plan, I think we should do it in the callbacks instead, but I'm happy to leave it as is because I think it fits nicely in https://github.com/DiamondLightSource/hyperion/issues/1217

Reply:
Do what in the callbacks? Create the points out of the parameters?

Reply:
yeah, or out of the grid scan info, whichever